rest/blip_api_test.go

```diff
@@ -134,9 +134,11 @@ func TestBlipPushRevisionInspectChanges(t *testing.T) {
 	receivedChangesRequestWg.Add(1)
 
 	// Wait until we got the expected callback on the "changes" profile handler
-	timeoutErr := WaitWithTimeout(&receivedChangesRequestWg, time.Second*15)
+	timeoutErr := WaitWithTimeout(&receivedChangesRequestWg, time.Second*5)
 	assertNoError(t, timeoutErr, "Timed out waiting")
-
+	log.Printf("Starting sleep")
```

Comment:
Can you add a comment explaining why this `time.Sleep()` is needed?   Is there any way to accomplish this w/ an event-based approach rather than a sleep based approach?  If not, might be worth mentioning why that's difficult, so it could later be converted.

Reply:
That was a legacy from some previous manual testing - I'll removed it.